### ISSUE TITLE
onRendered callback is not called after cell/value updates

### ISSUE THREAD

**Issue by Nyut0n** [olifolkerd/tabulator#1660]
Hey @olifolkerd , 
sorry I have to keep poking you with this. But I discovered a follow-up issue after you've fixed #1643

**Describe the bug**
The onRendered callback in a formatter function is called only initially. It doesn't run when a cell is updated.

**Tabulator Info**
- Tabulator 4.1.4
- Constructor, see below

**To Reproduce**
I've updated my fiddle from #1643 to use the latest 4.1.4 build and added a console.log to the onRendered callback of the num column: https://jsfiddle.net/htLj5z7a/11/
As you can see, the spinner builds perfectly at first, but after value updates it breaks, and that's apparently because the onRendered callback runs only once.

**Expected behavior**
Since the formatter runs with every value update, the onRendered, which is defined in the formatter, should run too.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version 71

**Comment by olifolkerd** [comment]
Hey @Nyut0n 

Good Spot! somehow that one had completely passed me by till now.

I will get a patch in for that in the next patch release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
I have pushed a fix for this to the master branch and will include it in this evenings patch release.

Cheers

Oli :)

**Comment by shaulnaim** [comment]
hi @olifolkerd  i tried even version 4.3.0 and still after changing cell the onRendered inside formatter is not getting called. it is working only on first table build.

what do i misse?
Thanks,
Shaul
